tango/text/convert/Layout.d

```diff
@@ -1000,7 +1000,7 @@ version (WithVariant)
 
 *******************************************************************************/
 
-package enum TypeCode
```

Comment:
It might be possible to use the fairly new protection syntax, `package(foo.bar)`, to avoid making the symbol public, but I chose to make it public for backwards compatibility.